python/src/SeticorePy/viewer.py

```diff
@@ -448,43 +451,67 @@ def show_correlations_text(self):
         for i in range(nants):
             print(f"{i:2d} " + " ".join(f"{corr[i, j]:.2f}" for j in range(nants)))
 
-            
-        
-def read_stamps(filename):
-    with open(filename) as f:
+
+def find_stamp_recipe(stamp_filepath, directory_path=None):
```

Comment:
here I also find "filepath" confusing - do you mean just the path component of the filename, or the entire filename of the stamp file?

Reply:
Right, I have it as `dir` + `filename` = `filepath`.

I think Python's [os.path](https://docs.python.org/3/library/os.path.html) agrees: 

```
os.path.abspath(path)
Return a normalized absolutized version of the pathname path...

os.path.basename(path)
Return the base name of pathname path. This is the second element of the pair returned by passing path to the function [split()](https://docs.python.org/3/library/os.path.html#os.path.split)...

os.path.dirname(path)
Return the directory name of pathname path. This is the first element of the pair returned by passing path to the function [split()](https://docs.python.org/3/library/os.path.html#os.path.split).

```